npm/metrics/ipsets.go

```diff
@@ -63,7 +63,13 @@ func RemoveEntryFromIPSet(setName string) {
 	}
 }
 
-// RemoveAllEntriesFromIPSet sets the number of entries for ipset setName to 0.
+// ResetEntriesInIPSet sets the number of entries for ipset setName to 0.
+// It doesn't ever update the number of IPSets.
+func ResetEntriesInIPSet(setName string) {
+	numIPSetEntries.Add(-getEntryCountForIPSet(setName))
```

Comment:
We shouldn't have to update prometheus. For memory consumption and scraping considerations, we currently don't track set names which have entry counts of 0. If we decide to change this later, we should bake that change into the existing functions (RemoveAllEntriesFromIPSet and RemoveEntryFromSet)

Reply:
Sounds good, let me remove metrics changes.